hypervisor/Makefile

```diff
@@ -1,7 +1,7 @@
 
 
 ASFLAGS = --32 -march=i486 -mtune=i486
-CFLAGS = -m32 -I../include
+CFLAGS = -m32 -fno-stack-protector -I../include
```

Comment:
why add this ?

Reply:
This will fix an issue for ubuntu env. 
Add this can let your code compile pass on ubuntu.